src/ctia/entity/actor.clj

```diff
@@ -153,5 +154,5 @@
    :realize-fn realize-actor
    :es-store ->ActorStore
    :es-mapping actor-mapping
-   :services->routes actor-routes
+   :services->routes #'actor-routes
```

Comment:
Curious, why var quoting?

Reply:
Since `actor-entity` is a top-level immutable data structure, it'll deref `actor-routes` only at compile-time. Reloads of `actor-routes` will not be seen.

By passing a var instead, calling `services->routes` will always see the latest value.

Ideally, `actor-entity` should eventually be a function. This is a halfway measure until that time.

Reply:
Interesting. Thank you for explaining it.

Reply:
thanks for the explanation. I assume that it's for helping in the repl?
I know that it would be repeated in each entity, but a comment on the code too could help and prevent future "regressions" on this.

Reply:
Great idea. I made a `reloadable-function` macro instead of the opaque `#'` syntax and added a docstring there instead.